trans/views/translation.py

```diff
@@ -108,12 +108,20 @@ def get(self, request, contest_slug, task_name):
                 print(type(e))
         else:
             logger.debug('Skipping monitor update')
-        return render(request, 'editor.html',
-                      context={'trans': trans.get_latest_text(), 'task': task_text,
-                               'text_font_base64': user.text_font_base64, 'contest_slug': contest_slug,
-                               'contests': contests, 'task_name': task_name, 'is_editor': user.is_editor(),
-                               'taskID': task.id, 'language_code': user.language.code, 'username': user.username,
-                               'direction': user.language.direction()})
+        return render(request, 'editor.html', context={
+            'trans': trans.get_latest_text(),
+            'task': task_text,
+            'text_font_base64': user.text_font_base64,
+            'contest_slug': contest_slug,
+            'contests': contests,
+            'task_name': task_name,
+            'is_editor': user.is_editor(),
+            'is_online': getattr(user, 'online'),
```

Comment:
This has come up multiple times. Can we create a helper method `is_online()`, similar to `is_editor()`?